### ISSUE TITLE
Different output on Docker

### ISSUE THREAD

**Issue by arielnmz** [wkhtmltopdf/wkhtmltopdf#4639]
**wkhtmltopdf version(s) affected**: 0.12.5

**OS information**
Host: Fedora 31
Docker: Alpine Linux 3.11

**Description**
Converting the same source document produces different output on my host vs docker

**How to reproduce**
Fedora (host): Just install wkhtmltopdf form the repos

`dnf install wkhtmltopdf`

Alpine (docker): Same

`apk add -U --no-cache wkhtmltopdf`

Generate a document on the host:

`wkhtmltopdf /tmp/vn/vn_1584497888387.pdf.html test2.pdf`

Then, generate a document on docker:

`docker run -it --rm --network host -v /tmp/vn/vn_1584497888387.pdf.html:/srcdoc.html -v "$(pwd)":/out vn_api_celery wkhtmltopdf /srcdoc.html /out/test3.pdf`

(I'm using host network mode because the doc loads some static css files)

The output on the second document being rendered incorrectly. Here's a comparison between the source HTML and the 2 generated docs

![image](https://user-images.githubusercontent.com/24735547/76920743-7fabdf80-6889-11ea-9b17-eff0c99a9ab4.png)

**Expected behavior**
The same output

**Possible Solution**  
I don't know if the Alpine build is missing something but providing prebuilt binaries would be a solution (maybe tarball releases or something?)

**Comment by Altenrion** [comment]
Also struggling with wkhtmltopdf in alpine docker build. Not printing svg charts. If i woud find smthing usefull, will post it here.

**Comment by markomarkovic** [comment]
You might be missing fonts. The host most likely has many installed, but the docker image might have none. I've had good success with this image: https://github.com/Surnet/docker-wkhtmltopdf#other-images

**Comment by arielnmz** [comment]
Makes sense, I'm no longer using it because of this, sadly. But will try again as soon as I pull this back up from my backlog

**Comment by thudxz** [comment]
I met a similar problem: the PDF can not display images with class attributes (from CSS file). Following the method from [here](https://gist.github.com/Rajeshr34/2e9b2438ff142e51c729b4b9b772680a ), I installed wkhtmltopdf with Patched QT Setup. It worked.
